packages/javascript/bh-shared-ui/src/components/HelpTexts/ADCSESC10b/LinuxAbuse.tsx

```diff
@@ -112,27 +130,32 @@ const LinuxAbuse: FC = () => {
         </>
     );
 
-    const step5 = (
+    const step6 = (
         <>
             <Typography variant='body2'>
-                <b>Step 5 (Optional): </b>Set <code>dNSHostName</code> of victim to the previous value.
+                <b>Step 6 (Optional): </b>Set <code>dNSHostName</code> and SPN of victim to the previous values.
                 <br />
                 <br />
-                To avoid DNS issues in the environment, set the <code>dNSHostName</code> of the victim computer back to
-                it's previous value using Certipy:
+                To avoid issues in the environment, set the <code>dNSHostName</code> and SPN of the victim computer back
+                to it's previous values using Certipy and ldapmodify:
```

Comment:
Sorry! I thought I marked each of these but I guess I forgot to save them? There's a few more instances of incorrect `it's`.